_temp/hud/package.json

```diff
@@ -26,21 +26,23 @@
     "browser-sync": "2.9.11",
     "chance": "^0.8.0",
     "css-loader": "0.20.2",
+    "del": "^2.2.0",
     "exports-loader": "0.6.2",
     "expose-loader": "0.7.0",
     "file-loader": "^0.8.4",
     "gulp": "3.9.0",
     "gulp-if": "2.0.0",
     "gulp-minify-html": "1.0.4",
     "gulp-util": "3.0.6",
+    "gulp-zip": "^3.2.0",
     "imports-loader": "0.6.5",
     "jscs": "2.3.4",
     "jshint": "2.8.0",
     "jshint-loader": "0.8.3",
     "lodash": "^3.10.1",
     "minimist": "1.2.0",
     "moment": "^2.10.6",
-    "node-sass": "3.3.3",
+    "node-sass": "3.4.2",
```

Comment:
I remember that I had issues with updating to later versions of `node-sass`, but I don't know if that was for just the client or if I had the problem with HUD as well... do you know if things still work ok?